src/lib/functions/form-submissions-handler.js

```diff
@@ -126,7 +126,13 @@ const createFormSubmissionHandler = function ({ functionsRegistry, siteUrl }) {
   }
 }
 
-// note: use Object.fromEntries post Nodejs 12.0.0
+/**
+ * Converts URLSearchParams to an object
+ * @deprecated can be replaced with `Object.fromEntries` once the support for older Node.js Versions than `12.0.0` is dropped.
+ * @param {URLSearchParams} urlParams 
+ * @returns {Record<string, any>}
+ */
+?**
```

Comment:
```suggestion
```

Reply:
Yep. Saw that and removed it in the commit. Thank you.